components/pagination/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@hods/pagination",
+  "version": "0.1.8",
+  "description": "Component to allow separating long tables into separate tables",
+  "main": "src/Pagination.tsx",
+  "sass": "assets/Pagination.scss",
+  "publishConfig": {
+    "main": "dist/Pagination.js",
+    "typings": "dist/Pagination.d.ts"
+  },
+  "files": [
+    "/assets",
+    "/dist"
+  ],
+  "scripts": {
+    "test": "jest",
+    "prepublishOnly": "npm run clean && npm run build",
+    "build": "tsc",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo"
+  },
+  "author": "Daniel A.C. Martin <npm@daniel-martin.co.uk> (http://daniel-martin.co.uk/)",
+  "license": "MIT",
+  "keywords": [
+    "react-components"
+  ],
+  "dependencies": {
+    "@hods/sass-base": "^0.1.10",
```

Comment:
This should have come from the workspace. It looks like you might have installed with `npm` instead of `pnpm`.